redis_views/response.py

```diff
@@ -5,12 +7,22 @@
 
 import redis
 
+logger = logging.getLogger(__name__)
```

Comment:
I think it might be better to embed the logger on the class because, according to some recommendations I've seen, it's better for third party libraries (like this one).

I consulted this site: http://victorlin.me/posts/2012/08/26/good-logging-practice-in-python in this section:
`Do not get logger at the module level unless disable_existing_loggers is False`